src/extension.ts

```diff
@@ -12,11 +12,12 @@ import {
 
 /* local */
 import { AnsiblePlaybookRunProvider } from './features/runner';
-
```

Comment:
Same: plz keep a two-line separator between imports and the module body.

Reply:
I fixed it but if you want this to be followed please make a change to eslint config to enforce it. I guess https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/newline-after-import.md should do the trick.